Dockerfile.ci

```diff
@@ -1030,7 +1030,7 @@ function start_api_server_with_examples(){
     airflow dags reserialize
     echo "Example dags parsing finished"
     echo "Create admin user"
-    airflow users create -u admin -p admin -f Thor -l Administrator -r Admin -e admin@email.domain
+    airflow users create -u admin -p admin -f Thor -l Administrator -r Admin -e admin@email.domain > /dev/null 2>&1 || true
```

Comment:
Same here. If there are errors we will not see them. That's a recipe for few hours of head scratching if we have a bug